backend/apps/owasp/templates/search/project.html

```diff
@@ -159,6 +159,11 @@ <h4 class="px-1">
         } = Vue;
         createApp({
             delimiters: ['${', '}'],
+            computed: {
+                searchPlaceholderText() {
+                    return `Search among ${this.projects.length} projects to contribute to...`;
```

Comment:
This is the right implementation. But it's a half-done -- we'd like to emphasize the total amount of projects OWASP has (somewhat around 235). So you could either hardcode that value or create another API endpoint and get this number from `Project::active_projects_count` (requires backend work and API endpoint naming thoughts)

Reply:
@arkid15r -- I have added an endpoint `/api/v1/owasp/search/active-projects-count` which just returns the `Project.active_projects_count()`. Figured this is also a "search" of sorts, so kept it in the existing `search.py`. 

Please let me know if the naming/location is not intuitive and more than happy to iterate on this :)